plugins/balloontoolbar/skins/default.css

```diff
@@ -24,6 +24,8 @@ For licensing, see LICENSE.md or https://ckeditor.com/license
 /* The .cke_balloon_content container height will be stretched to children height, so padding is properly applied. */
 .cke_balloon.cke_balloontoolbar .cke_balloon_content {
```

Comment:
This style is duplicated (with different `padding` value) few lines below (https://github.com/ckeditor/ckeditor-dev/pull/1514/files#diff-b5032182871f964b74b65d3e1f7c187fL54).